Demo/Sources/YouTube/YouTubeMetadata.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import Player
+import UIKit
+
+struct YouTubeMetadata: AssetMetadata {
+    let image: UIImage
+
+    func nowPlayingMetadata() -> NowPlayingMetadata {
+        .init(image: image)
```

Comment:
We could eventually display some title, maybe the video id or _YouTube_ (or a combination of both) until YouTubeKit provides more info.

Reply:
Good idea. 👌